client/my-sites/plans/plan-overview/plan-feature/index.jsx

```diff
@@ -0,0 +1,40 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+import CompactCard from 'components/card/compact';
+
+const PlanFeature = ( { button, description, heading } ) => {
+	return (
+		<CompactCard className="plan-feature">
+			<div>
+				<strong className="plan-feature__heading">{ heading }</strong>
+				<span>{ description }</span>
+			</div>
+			{ button &&
+				<Button
+					className="plan-feature__button"
+					href={ button.href }
```

Comment:
Talking a bit with @breezyskies... can we try a `compact` button here? I think it'd work a bit better in terms of weight.